coral/src/app/features/approvals/topics/components/TopicApprovalsTable.tsx

```diff
@@ -0,0 +1,140 @@
+import {
+  Box,
+  DataTable,
+  DataTableColumn,
+  GhostButton,
+  Icon,
+} from "@aivenio/aquarium";
+import deleteIcon from "@aivenio/aquarium/dist/src/icons/delete";
+import tickCircle from "@aivenio/aquarium/dist/src/icons/tickCircle";
+import infoSign from "@aivenio/aquarium/dist/src/icons/infoSign";
+import { TopicRequestNew } from "src/domain/topic/topic-types";
+
+interface TopicRequestTableData {
+  id: number; // unclear
+  topicname: string;
+  environmentName: string;
+  topictype: string;
+  teamname: string; // unclear
+  requestor: string;
+  requesttimestring: string;
+}
+
+const columns: Array<DataTableColumn<TopicRequestTableData>> = [
+  { type: "text", field: "topicname", headerName: "Topic" },
+  { type: "text", field: "environmentName", headerName: "Cluster" },
+  { type: "text", field: "topictype", headerName: "Type" },
+  { type: "text", field: "teamname", headerName: "Claim by team" },
+  { type: "text", field: "requestor", headerName: "Requested by" },
+  {
+    type: "text",
+    field: "requesttimestring",
+    headerName: "Date requested",
+    formatter: (value) => {
+      return `${value} UTC`;
+    },
+  },
+  {
+    type: "custom",
+    // @TODO PR open in DS to be able to
+    // add an "invisible" header name that
+    // is used as aria-label. That will also
+    // solve the duplicate key warning
+    //https://github.com/aiven/design-system/pull/950
+    headerName: "Details",
+    width: 30,
+    UNSAFE_render: (row) => {
+      return (
+        <GhostButton
+          onClick={() => alert("Approve")}
+          title={`View topic request `}
+          dense
+        >
+          <Box
```

Comment:
Probably better to use the plain `GhostButton` like in the other PR ^^

Reply:
Oh gosh, I thought I updated that already. Sorry. Will change!